src/devices/DCMotor/README.md

```diff
@@ -9,8 +9,5 @@ DC motors are controlled by simply providing voltage on the inputs (inverted vol
 DC motors can be controlled with 1, 2 or 3 pins.
 Please refer to the [sample](samples/README.md) to see how to connect it.
 
-**Recommended setup is 3-pin mode, 1 and 2 pin mode may cause excessive heat or damage your H-bridge (larger engines are more likely to damage).**
```

Comment:
It would be nice to explain a bit more why is 3 pin better than 2. From what I can tell in the code and on the README it is not clear why I should pick one over the other.

Reply:
do you mean to expand `1 and 2 pin mode may cause excessive heat or damage your H-bridge`? H-bridges are designed such that inputs should not change that quickly

Reply:
yes, I mean that we should elaborate a bit more on that as in, why is 3 pin better, where are they used, how are the instructions passed to the device different, etc.